src/scripts/growth-hack.ts

```
@@ -18,60 +18,69 @@ interface RepoInfo {
 }
```

```diff
+    } catch (error) {
```

Catching and logging errors is good practice, but consider rethrowing or handling them in a way that doesn't silently fail in production environments.